src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Extensions/SymbolInfoExtensions.cs

```diff
@@ -16,15 +16,15 @@ public static ImmutableArray<ISymbol> GetAllSymbols(this SymbolInfo info)
             => GetAllSymbolsWorker(info).Distinct();
 
         private static ImmutableArray<ISymbol> GetAllSymbolsWorker(this SymbolInfo info)
-            => info.Symbol == null ? info.CandidateSymbols : [info.Symbol, .. info.CandidateSymbols];
+            => info.Symbol == null ? info.CandidateSymbols : info.CandidateSymbols.Insert(0, info.Symbol);
 
         public static ISymbol? GetAnySymbol(this SymbolInfo info)
             => info.Symbol ?? info.CandidateSymbols.FirstOrDefault();
 
         public static ImmutableArray<ISymbol> GetBestOrAllSymbols(this SymbolInfo info)
         {
             if (info.Symbol != null)
-                return [info.Symbol];
+                return ImmutableArray.Create(info.Symbol);
```

Comment:
Why did we need to undo collection expressions here?

Reply:
Not 100% sure. Source build was failing.

Reply:
The source build in CI or the full source build?

Reply:
CI